### PR TITLE
Decode Morse message

### PR DESCRIPTION
**Pull request by GiuseppeTG**
### Project requirements
- Create a method to decode a Morse code character, takes a string parameter, and return the corresponding character in uppercase (e.g. `decode_char(".-")` returns `"A"`).
- Create a method to decode an entire word in Morse code, takes a string parameter, and return the string representation. Every character in a word will be separated by a single space (e.g. `decode_word("-- -.--")` returns `"MY"`).
- Create a method to decode the entire message in Morse code, takes a string parameter, and return the string representation. Every word will be separated by 3 spaces (e.g.
    ```
      decode("-- -.--   -. .- -- .")
    ```
  
  returns `"MY NAME"`).
- Reuse methods whenever possible. Make sure that your code is DRY.
- Once your code is ready use it to decode the message from the old bottle:

```
      .-   -... --- -..-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,67 @@
...
+
+![gh actions failing checks](../assets/images/gh-actions-rubocop-failing-checks.png)
+
+## [OPTIONAL]Set-up RSpec GitHub Action
+
+You can run your tests with GitHub Actions to ensure that they are passing before merging a PR.
+
+To use the GitHub Action to run your tests, please do the following **steps in this order**:
+
+1. Add a copy of [`.github/workflows/tests.yml`](.github/workflows/tests.yml) to your `.github/workflows` folder.
```
Comment (omar-muhamad) [inline README.md]: - [ ] Please edit your README file by using the [Microvere README template](https://github.com/microverseinc/readme-template/blob/master/README.md) and edit it with your project and personal data to make it more professional.

File: morsecode.rb [hunk 2]
```diff
@@ -1,3 +1,20 @@
...
+end
+
+def convert_string(string)
+  arr = []
+  words = string.split('   ')
+  words.each { |word| arr.push(convert_word(word)) }
+  arr = arr.join(' ')
+end
+
+puts convert_string('.-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-.   .-. ..- -... .. . ...')
```
Comment (omar-muhamad) [inline morsecode.rb]: - [ ] Please add a method to decode a Morse code character, takes a string parameter, and return the corresponding character in uppercase. as mentioned in the project description.

image

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi Team,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Gitflow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by AshakaE** [comment]
Hi team , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**